content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -185,7 +185,7 @@ another claim because the previous claimant's data remains on the volume.
 An administrator can manually reclaim the volume with the following steps.
 
 1. Delete the PersistentVolume. The associated storage asset in external infrastructure
-   (such as an AWS EBS or GCE PD volume) still exists after the PV is deleted.
+   (such as a GCE PD volume) still exists after the PV is deleted.
```

Comment:
GCE PD in-tree volume plugin was removed in 1.28. Do we still need this in the doc on the main branch?

Reply:
Removed GCE PD doc in this file.